src/screens/AccountUnlock.js

```diff
@@ -95,10 +113,13 @@ export class AccountUnlock extends React.PureComponent {
 											routeName: 'LegacyAccountList'
 										}),
 										NavigationActions.navigate({ routeName: 'AccountDetails' }),
-										NavigationActions.navigate({ routeName: next })
+										NavigationActions.navigate({
+											params: next === 'AccountBiometric' ? { pin: pin } : {},
```

Comment:
`next` wouldn't be 'AccountBiometric', there is already a `if` sentences before.